theforeman.org/pipelines/infra/updateJobs.groovy

```diff
@@ -14,7 +14,7 @@ pipeline {
                 ])
 
                 virtEnv('jjb-venv', 'pip install pbr')
-                virtEnv('jjb-venv', 'pip install jenkins-job-builder')
+                virtEnv('jjb-venv', 'pip install jenkins-job-builder<5')
```

Comment:
you could argue, this should be a `-r requirements.txt` instead?

Reply:
Indeed. I'd prefer that.